spec.html

```diff
@@ -39963,7 +39962,7 @@ <h1>Map ( [ _iterable_ ] )</h1>
       <emu-clause id="sec-add-entries-from-iterable" type="abstract operation">
         <h1>
           AddEntriesFromIterable (
-            _target_: unknown,
+            _target_: an ordinary object,
```

Comment:
is this worth calling out over just saying `an Object`?

Reply:
Hard to say if the benefit is worth the cost, because they're both fairly small (I think). I'm okay either way, whatever the editors want.

Reply:
@jmdyck Decision from editor call was to not use "an ordinary object" as a parameter type unless that refinement was needed for the algorithm steps (which it isn't here).